protostar/starknet/cheatable_execute_entry_point.py

```diff
@@ -70,7 +70,7 @@
 @dataclass(frozen=True)
 class ContractProfile:
     contract_callstack: list[ContractFilename]
-    entry_point: ContractEntryPoint
+    # entry_point: ContractEntryPoint
```

Comment:
@MaksymilianDemitraszek this can be removed probably, but left commented for you to review since it's profiler-specific